relational_operators/HashJoinOperator.hpp

```diff
@@ -265,7 +324,9 @@ class HashInnerJoinWorkOrder : public WorkOrder {
         selection_(selection),
         hash_table_(hash_table),
         output_destination_(DCHECK_NOTNULL(output_destination)),
-        storage_manager_(DCHECK_NOTNULL(storage_manager)) {}
+        storage_manager_(DCHECK_NOTNULL(storage_manager)) {
+    preferred_numa_nodes_.push_back(numa_node);
```

Comment:
Question: in a machine w/o `libnuma`, what does `0` mean in `preferred_numa_nodes_`? Thanks.